test/ecto/adapters/postgres_test.exs

```diff
@@ -69,6 +71,19 @@ defmodule Ecto.Adapters.PostgresTest do
     IO.iodata_to_binary SQL.delete(prefx, table, filter, returning)
   end
 
+  setup meta do
```

Comment:
We don't need those tests at the adapter level. The unit tests you added in migration_test is enough.

Reply:
Please see the comment reply for mysql_test.exs above.